docs/diagnostics/PX1036.md

```diff
@@ -8,18 +8,18 @@ This document describes the PX1036 diagnostic.
 | PX1036 | The DAC primary key should be named `PK`. The class containing DAC foreign keys should be named `FK`.  | Warning (ISV Level 3: Informational) | Available | 
 
 ## Diagnostic Description
-The diagnostic checks the correct naming of a primary key field declarations in DACs. For a primary key, the name should be `PK`.
+The diagnostic checks the correct naming of a primary and foreign key field declarations in DACs. For a primary key, the name should be `PK`. For a foreign key, the name should be `FK`.
 The warning is shown only for DACs that have the `PXCacheName` or `PXPrimaryGraph` attribute.
 The diagnostic does not check abstract DACs and DACs which have only unbound DAC properties (fully-unbound DACs used for custom popups and filters for inquiry forms).
 
 To fix the issue, change the name of the primary key class to `PK`, and change the name of the class that contains foreign key classes to `FK`.
 
-The code fix suggests that the developer changes the name of the primary key class to `PK`.
+The code fix suggests that the developer changes the name of the primary key class to `PK`, and the name of the foreign key to `FK`.
```

Comment:
This is incorrect. For foreign keys the fix will search for public static FK class and there are three possible outcomes:
- The FK class is not found. Then it is generated and **all** DAC foreign keys are moved there
- The FK class is found and it is public and static. Move all DAC foreign keys declarations which are not already inside FK class into it.
- The FK class is found but it's not declared correctly (not public or not static). No code fix in this case

Reply:
No code fix description for UK